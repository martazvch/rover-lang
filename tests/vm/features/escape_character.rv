fn main() {
    print "\nIm on next line\n"
    print "Not escaped \\n"
    print "Here are some quotes: one \" two \""
}
