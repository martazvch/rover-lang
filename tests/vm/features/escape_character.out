
Im on next line
Not escaped \n
Here are some quotes: one " two "

